env/bicep/pr/01_sql/02_deployment/parameters.bicepparam

```diff
@@ -0,0 +1,8 @@
+using './../../../../../IAC/Bicep/bicep/01_sql/02_deployment/main.bicep'
+
+param sqlServerAdministratorLogin = readEnvironmentVariable('sqlServerAdministratorLogin')
+param sqlServerAdministratorPassword=readEnvironmentVariable('sqlServerAdministratorPassword')
+param environment= 'pr'
```

Comment:
update all the env param value to readRnvironmentVariable(). you will then get rid of the iac.biceo-pr file and the calls to it from the workflow tasks.

Reply:
done